vars/loadResources.groovy

```diff
@@ -1,17 +1,24 @@
+import io.openshift.Utils
+
 def call(Map args = [:]) {
-  if(!args.file) {
+  if (!args.file) {
     error "Missing manadatory parameter: file"
   }
 
   def yaml = readYaml(file: args.file)
+  def validate = args.validate ?: "true"
```

Comment:
`true` instead of `"true"` . Groovy has support of boolean just like Java

Reply:
changed to boolean